library/components/core/NetworkClient.bs

```diff
@@ -33,9 +33,7 @@ function postFromFile(url as string, filePath as string, headers as object, payl
     m.urlTransfer.SetRequest("POST")
     m.urlTransfer.SetCertificatesFile("common:/certs/ca-bundle.crt") ' required for SSL
 
-    for each header in headers
-        m.urlTransfer.AddHeader(header, headers[header])
-    end for
+    m.urlTransfer.SetHeaders(headers)
```

Comment:
We're not doing anything with the Boolean that's returned.
Now sure how we could though.